src/app/zap-templates/zcl/data-model/chip/wake-on-lan-cluster.xml

```diff
@@ -24,6 +24,10 @@ limitations under the License.
     <client init="false" tick="false">true</client>
     <server init="false" tick="false">true</server>
     <description>This cluster provides an interface for managing low power mode on a device that supports the Wake On LAN protocol.</description>
-    <attribute side="server" code="0x0000" define="WAKE_ON_LAN_MAC_ADDRESS" type="char_string" length="32" writable="false" optional="true">MACAddress</attribute>
+
+    <global side="either" code="0xFFFD" value="1"/>
+
+    <attribute side="server" code="0x0000" define="WAKE_ON_LAN_MAC_ADDRESS" type="char_string" length="12" writable="false" optional="true">MACAddress</attribute>
+    <attribute side="server" code="0x0001" define="LINK_LOCAL_ADDRESS" type="octet_string" length="16" writable="false" optional="true">LinkLocalAddress</attribute>
```

Comment:
This thing is clearly provisional, since just as clearly there was no certification done for it...  Is there a spec issue tracking marking it provisional?  Should it be marked provisional here?  Or do we not have a way to do that for attributes yet?

Reply:
ZAP support for provisional attributes exists (https://github.com/project-chip/zap/commit/1642f1be28926f8b3641cf00d7aa3961e9abfe84), however it is not yet supported by our generators to use it. Can add it to zap xml though.